server/src/test/resources/csars/yaml/valid/lamp-csar-no-input/apache_web_server/webserver_install.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+sudo apt-get install apache2 -y
+sudo apt-get install libapache2-mod-php7.0 
```

Comment:
i think you have to add `-y` here too.

You could also consider concatenation of both commands like that:
`sudo apt-get install -y apache2 libapache2-mod-php7.0`